frontend/src/ui/Blocks/Series.tsx

```diff
@@ -208,6 +232,32 @@ const VideoGrid: React.FC<React.PropsWithChildren> = ({ children }) => (
     </div>
 );
 
+const UpcomingEventsGrid: React.FC<React.PropsWithChildren> = ({ children }) => {
+    const { t } = useTranslation();
+
+    return <details css={{
+        backgroundColor: "var(--grey86)",
+        borderRadius: 4,
+        "> summary": {
+            color: "var(--grey20)",
+            cursor: "pointer",
+            fontSize: 14,
+            padding: "6px 12px",
+        },
+        "> hr": {
+            margin: "0 12px",
+        },
```

Comment:
I would:

- Add some margin bottom to the `details` (8 or 12 maybe?)
- Add some space between the triangle and the text in `<summary>`